packages/react-core/src/components/ApplicationLauncher/ApplicationLauncher.tsx

```diff
@@ -63,6 +63,8 @@ export interface ApplicationLauncherProps
   toggleId?: string;
   /** @beta Opt-in for updated popper that does not use findDOMNode. */
   removeFindDomNode?: boolean;
+  /** z-index of the application launcher */
+  zIndex?: number;
```

Comment:
This would apply to most of the other components as well, but for the description it'd be worth mentioning that this prop would only apply when `menuAppendTo` is not inline.

I left additional comments where this might be different.

Reply:
Gotcha thanks for pointing them out, I had forgotten that the dropdown components didn't always use Popper